pkg/actors/actors.go

```diff
@@ -332,6 +326,8 @@ func (a *actorsRuntime) callLocalActor(ctx context.Context, req *invokev1.Invoke
 		return nil, fmt.Errorf("error from actor service: %s", string(respData))
 	}
 
+	diag.DefaultMonitoring.ActorActivated(actorTypeID.GetActorType())
```

Comment:
This will trace an actor activation on every call. Since Dapr doesn't know now if the actor has been activated or not, we probably shouldn't consider this a trace event.